test/models/concerns/association_test_module.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+module AssociationTestModule
+  extend ActiveSupport::Concern
+
+  class_methods do
+    # rubocop:disable Naming/PredicateName
+    def has_many(associations, object = nil)
+      object ||= to_s.delete_suffix('Test')
+
+      [associations].compact.flatten.each do |association|
```

Comment:
Burada döngüye gerek yok gibi.
```ruby
def has_many(association, object = nil)
  object ||= to_s.delete_suffix('Test')

  test "#{object} has_many #{association}" do
    assert object.constantize.take.send(association)
  end
end
```

Reply:
One-liner ve çoklu attribute senaryolarını birlikte desteklesin diye yaptım bunu. Örneğin:

```ruby
has_many :addresses
```

ve

```ruby
has_many(%i[addresses identities student_numbers])
```

gibi. Her ikisi de geçerli şuanki haliyle.